examples/free_tier_appservice/terraform.template.tfvars

```
@@ -0,0 +1,16 @@
+auth_token      = "v4-api-key-secret"
+organization_id = "<organization_id>"                            
+project_id      = "<project_id>"
+cluster_id      = "<cluster_id>"
+
+
+app_service = {
+  name        = "new-terraform-app-service modified"
```

This name is not appropriate
```suggestion
  name        = "new-terraform-free-tier-app-service"
```

---

changed here 4861e37e3ea8db3acedf994d7fa7a7f249d4539a